engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -181,6 +186,11 @@ class WorkflowManagerActor(config: Config,
       data.idFromActor(workflowActor) foreach { workflowId =>
         jobStoreActor ! RegisterWorkflowCompleted(workflowId)
         workflowStore ! WorkflowStoreActor.RemoveWorkflow(workflowId)
```

Comment:
Won't the workflow already be removed from the WorkflowStore in the abort case here ?